sw/device/tests/dif/dif_plic_unittest.cc

```diff
@@ -99,115 +117,79 @@ class IrqTests : public PlicTest {
   }
 };
 
-class IrqEnableSetTest : public IrqTests {
- protected:
-  // Make sure our tests are up to date.
-  IrqEnableSetTest() {
-    EXPECT_EQ(RV_PLIC_PARAM_NUMTARGET, 1);
-    EXPECT_EQ(registers_.size(), RV_PLIC_IE0_MULTIREG_COUNT);
```

Comment:
What happened with these checks?

Reply:
The first one was deleted completely by mistake, but the second is actually asserted by the length of the array constants. =)